app.js

```diff
@@ -0,0 +1,29 @@
+
+
+// jquery container doc ready
+$(() => {
+
+
+  // new background with each new tab
+  // rnadom
+  $("html").css({
+    "background-image": "url('https://source.unsplash.com/random/3840x2160')",
+    "background-repeat": "no-repeat",
+    "background-size": "cover"
+  });
+
+  //set blank quote and author; global
+  let quote = "";
+  let author = "";
+
+  // random get request from quote garden, replaces html in retrievedQuote div
+  $.get("https://quote-garden.herokuapp.com/api/v2/quotes/random", (data) => {
+    quote = data.quote.quoteText;
+    author = data.quote.quoteAuthor;
+    let newQuote = $('<p>' + '"' + quote + '"'+ '</p>' + '<h3>' + '-- ' + author + '</h3>');
```

Comment:
I would Recommend String Literal Formatting when you are building out lines like this as it helps reduce the necessity of + ""
example: ("The" + localString + "danced outside for" + localTimeCollected); 
Vs
` The ${LocalStrong} danced outside for ${localTimeCollected}`

Ps: This example is using Javascript Sting Literal Formatting in this example. and it also does work with HTML inside the string.

Reply:
Thank you! That is a great point!

Reply:
agree